+dj/Schema.m

```diff
@@ -7,7 +7,7 @@
     properties(SetAccess = private)
         package    % the package (directory starting with a +) that stores schema classes, must be on path
         dbname     % database (schema) name
-        prefix=''  % optional table prefix, allowing multiple schemas per database
+        prefix=''  % optional table prefix, allowing multiple schemas per database -- remove this feature if not used
```

Comment:
Let's talk about this feature - I have never seen it being used and it may be an unnecessary complexity.